grpc.gemspec

```diff
@@ -27,7 +27,7 @@ Gem::Specification.new do |s|
   s.require_paths = %w( src/ruby/bin src/ruby/lib src/ruby/pb )
   s.platform      = Gem::Platform::RUBY
 
-  s.add_dependency 'google-protobuf', '~> 3.1.0'
+  s.add_dependency 'google-protobuf', '~> 3.1'
```

Comment:
'~> 3.1' means ">= 3.1 and < 4.0". I don't actually know what would happen if you used '~> 3'.